boards/shields/huatian_mt9m114/huatian_mt9m114.overlay

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	chosen {
+		zephyr,camera = &csi;
+	};
+};
+
+&zephyr_cam_i2c {
+	mt9m114: mt9m114@48 {
+		compatible = "aptina,mt9m114";
+		reg = <0x48>;
+
+		port {
+			mt9m114_ep_out: endpoint {
+				remote-endpoint = <&csi_ep_in>;
+			};
+		};
+	};
+};
+
+&csi {
```

Comment:
Perhaps we should label this as `nxp_csi`? These properties aren't generic between all CSI devices in Zephyr, are they?

Reply:
The properties are generic. BTW, the name _csi_ is specific to NXP (and also not a well chosen name at all as it is confused with "MIPI CSI"). I think we should label it "zephyr_sensor_interface" so that all boards / vendors who have the same camera pipeline "mt9m114 -> sensor_interface" can reuse this overlay.

Reply:
Anyways, changed as your suggestion. I also fixed a typo:
`Pins assignment of the Huatian MT9M114 camera module
======================================`

Is it ok for you now ?